File: 프로그래머스/unrated/120904. 숫자 찾기/숫자 찾기.cpp
#include <string>
#include <vector>

using namespace std;

int solution(int num, int k) {
    int answer = 0;
    
    string num_String=to_string(num);
    string a=to_string(k);
    
    for(int i=0;i<num_String.length();i++)
    {
        if(num_String[i]==a[0]) 
        {
            answer=i+1;
            break;
        }
        else
            answer=-1;
    }
    
    return answer;
}

File: 프로그래머스/0/120904. 숫자 찾기/숫자 찾기.py
def solution(num, k):
    # Convert the number to a string to iterate through the digits
    for idx, digit in enumerate(str(num)):
        # Check if the current digit matches k
        if digit == str(k):
            return idx + 1  # Return 1-based index
    # If the digit was not found, return -1
    return -1
